Dockerfile

```diff
@@ -144,6 +144,38 @@ RUN \
         /usr/src/telldus \
         /usr/src/telldus-fix-gcc-11-issues.patch \
         /usr/src/telldus-fix-alpine-3-17-issues.patch
+
+# libgpiod Python bindings built against our Python 3.11
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
+RUN \
+    apk add --no-cache --virtual .build-dependencies \
+        build-base \
+        autoconf \
+        automake \
+        libtool \
+        linux-headers \
+        autoconf-archive \
+        python3-dev \
+        doxygen \
+        help2man \
+    && LIBGPIOD_VERSION=$(apk info -e -v libgpiod | awk -F '-' '{print $(NF-1)}') \
```

Comment:
We should specify the version over build.yaml

Reply:
`libgpiod` is currently installed from Alpine's repositories at the top of the `Dockerfile`, so I wanted to make the Python bindings match the package version we currently have installed as otherwise an Alpine version bump would desynchronize the two.

Should I remove the `apk add libgpiod` at the top and rewrite the stage to install the entire library here? Or do you still want to manually bump the version?

Reply:
We bundle libs in our Wheels packages, so IMHO we should just build wheels the normal route.